src/daemon/configuration.proto

```diff
@@ -1,13 +1,17 @@
 import "base/base.proto";
+import "net/universal.proto";
 
 package dist_clang.daemon.proto;
 
 message Host {
   required string host          = 1;
   optional uint32 port          = 2 [ default = 6000 ];
+  // Ignored for coordinators and collectors.
```

Comment:
Put the comments below the commented line and insert empty lines for better visualization.